blog/_posts/2016-08-15-Introducing-Team-Fedex++.md

```diff
@@ -50,34 +50,34 @@ We are Python lovers. We get a problem and the first way that comes in our mind
 * All our Coaches form a highly enthusiastic group of programmers who never fail to encourage us, are available 24/7 for help and are more than willing to motivate us to continue our journey in Open Source that we have started with RGSoC.
 
 
-![Team-Fedex++ at Work](/img/blog/2016/TeamFedex++3.jpeg)
+![Team-Fedex++ at Work](/img/blog/2016/TeamFedex++3.jpeg)    
 *image taken by Sahar Siddiqui* 
 
 
 ### **What we have achieved so far?**
-* We designed the basic draft of PyDSA visualisation website. Link to wireframes : <https://goo.gl/8paaNx>
-* We learnt about Pep8 format, Python Tests, Travis CI and a few issues and their solutions.
-* We set up a Django server and made a Test App in it. 
-* As guided by our coaches, we started learning more about d3.js library and tried to code a few visualisations in it. 
-* We learnt how does the interaction between the frontend and backend works using d3.js in frontend and Django framework.
-* We learnt about classes and objects in Python and Design Patterns that should be implemented in order to structure the PyDSA library.
-* Completed the list of algorithms mentioned in the list of algorithms required for the first official release.   
-
-In the process, we have reviewed some Pull Requests made by others and we have sent a few of them ourselves.
-* <https://github.com/pydsa/pydsa/issues/107> 
-* <https://github.com/pydsa/pydsa/pull/116>
-* <https://github.com/pydsa/pydsa/pull/120>
-*<https://github.com/pydsa/pydsa/pull/116/commits/3d4e52114e89eae33310a379f6031a955b9d19d2>
+* We designed the basic draft of PyDSA visualisation website. Link to wireframes : <https://goo.gl/8paaNx>  
+* We learnt about Pep8 format, Python Tests, Travis CI and a few issues and their solutions.  
+* We set up a Django server and made a Test App in it.   
+* As guided by our coaches, we started learning more about d3.js library and tried to code a few visualisations in it.    
+* We learnt how does the interaction between the frontend and backend works using d3.js in frontend and Django framework.   
+* We learnt about classes and objects in Python and Design Patterns that should be implemented in order to structure the PyDSA library.   
+* Completed the list of algorithms mentioned in the list of algorithms required for the first official release.     
+
+In the process, we have reviewed some Pull Requests made by others and we have sent a few of them ourselves.   
+* <https://github.com/pydsa/pydsa/issues/107>   
+* <https://github.com/pydsa/pydsa/pull/116>   
+* <https://github.com/pydsa/pydsa/pull/120>    
+* <https://github.com/pydsa/pydsa/pull/116/commits/3d4e52114e89eae33310a379f6031a955b9d19d2>    
```

Comment:
would it be possible to remove this line? It's redundant, as you link to the same pull request a couple of lines above, and the line is extremely long. alternatively, please use a title / sentence that links to the url of the commit instead.